docs/source/nextflow-workflow.rst

```diff
@@ -135,6 +135,143 @@ Example ``sample_sheet.csv``
+
+Autometa can bin one or several metagenomic datasets in one run. Regardless of the number of metagenomes you want to process, you will need to provide a sample sheet which specifies the name of your sample, the full path to where that data is found. If the metagenome was assembled via SPAdes, Autometa can extract coverage and contig length information from the sequence headers. If you used a different assembler you will need to provide either raw reads for coverage calculation or a table of contig/scaffold coverage. Full details for data preparation can be `found here <https://autometa.readthedocs.io/en/latest/nextflow-workflow.html#sample-sheet-preparation>`_
+
+First ensure that your Autometa conda environment is activated. You can activate your environment by running:
+
+.. code-block:: bash
+    conda activate autometa-nf
+
+Navigate to where your metagenome files are and run the following code to launch Autometa:
+
+.. code-block:: bash
+
+    nf-core launch KwanLab/Autometa
+    
+You will now use the arrow keys to move up and down between your options and hit your "Enter" or "Return" key to make your choice.
+
+Menu 1:
+The double, right-handed arrows should already indicate "2.0.0". This is the latest version of Autometa, hit your "Enter" to chose this option.
+
+.. image:: ../img/Menu1.png
+
+Menu 2:
+Pick the commandline option. Unless you've tunneled into your server, or are using Autometa locally, this is your best option.
+
+.. image:: ../img/Menu2.png
+
+Menu 3:
+If you are using a scheduler (Slurm in this example) the only thing you'll need to change is the profile option. If you are not using a scheduler, leave this blank.
+
+.. image:: ../img/Menu3.png
+
+Menu 4:
+Now we need to give Autometa the full path to where to find our sample sheet, where to put the output (it will generate a folder per metagenome in this folder) and where to store all the log files (aka the trace files)
+
+.. image:: ../img/Menu4.png
+
+Menu 5:
+This is where you can change your binning parameters. If you're not sure what you're doing I would recommend only changing the "length_cutoff". 
+The default cutoff is 3000bp, which means that any contigs/scaffolds smaller than 3000bp will not be considered for binning. This cutoff will depend on how good your assembly is: E.g. if your N50 is 1200 bp, I would chose a cutoff of 1000. If your N50 is more along the lines of 5000, I would leave the cutoff at the default 3000. I would strongly recommend against choosing a number below 900 here. In the example below, I have chosen a cutoff of 1000bp as my assembly wasn't particularly great
+
+.. image:: ../img/Menu5.png
+
+Menu 6:
+Here you have a choice to make:
+By enabling taxonomy aware mode, Autometa wll attempt to use taxonomic data to make your bins more accurate. However, this is a more computational expensive step and will make the process take longer. By leaving this option as the default "False" option, Autometa will bin according to kmer patterns.
```

Comment:
Can you break this up into multiple lines to avoid the text-wrapping?

```suggestion
By enabling taxonomy aware mode, Autometa will attempt to use taxonomic data to make your bins more accurate. However, this is a more computationally expensive step and will make the process take longer. By leaving this option as the default "False" option, Autometa will bin according to coverage and kmer patterns.
```